### PR TITLE
1st...

### PR DESCRIPTION
**Pull request by radscheit**
#413

### CODE REVIEW

File: app/views/events/participants.html.erb [hunk 1]
```diff
@@ -106,30 +106,38 @@
...
-  <% if @event.phase == :execution %>
-    <%= link_to t('events.badges.print_button_label'),
-                badges_event_path(@event),
-                class: 'btn btn-default',
-                disabled: @participants.empty? %>
-  <% end %>
+      <div class="btn-group">
+        <button class="btn btn-default btn-sm" type="button" data-toggle="modal" data-target="#print_participant_modal" id="open_print_modal">Teilnehmerliste (PDF)</button>
+        <% if can? :print_applications_eating_habits, Event %>
+        <%= link_to t('events.participants.print_habits'),
```
Comment (Blaidd-Drwg) [inline app/views/events/participants.html.erb]: Indentation

File: app/views/events/_participants_modal.html.erb [hunk 2]
```diff
@@ -23,7 +23,10 @@
       </div>
       <div class="modal-footer">
         <button type="button" class="btn btn-default" data-dismiss="modal"><%= t('.close_modal') %></button>
-        <%= submit_tag(t('.create_pdf'), class: 'btn btn-default', id:'print_participant_list' ) %>
+        <%= submit_tag(t('.create_pdf'), 
+            class: 'btn btn-default', 
+            id:'print_participant_list',
+            :formtarget => "_blank" ) %>
```
Comment (Blaidd-Drwg) [inline app/views/events/_participants_modal.html.erb]: Why do you use `formtarget` here and `target` in the other `submit_tag`?
Also, mixing the two syntaxes for hashes looks weird, please keep it consistent.

File: app/views/events/participants.html.erb [hunk 3]
```diff
@@ -106,30 +106,38 @@
...
-              event_path(@event), :class => 'btn btn-default'  %>
-
-  <% if @event.phase == :execution %>
-    <%= link_to t('events.badges.print_button_label'),
-                badges_event_path(@event),
-                class: 'btn btn-default',
-                disabled: @participants.empty? %>
-  <% end %>
+      <div class="btn-group">
+        <button class="btn btn-default btn-sm" type="button" data-toggle="modal" data-target="#print_participant_modal" id="open_print_modal">Teilnehmerliste (PDF)</button>
```
Comment (Blaidd-Drwg) [inline app/views/events/participants.html.erb]: Please I18n-ify this while you're at it :wink:

**Review by Blaidd-Drwg — CHANGES_REQUESTED** [review]

File: app/views/events/_participants_modal.html.erb (hunk 2 above)
Comment (radscheit) [reply app/views/events/_participants_modal.html.erb]: formtarget follows the specific syntax description for submit tag

Comment (Blaidd-Drwg) [reply app/views/events/_participants_modal.html.erb]: The `submit_tag` in `app/views/events/participants.html.erb` has the normal target though :confused:

Comment (radscheit) [reply app/views/events/_participants_modal.html.erb]: OK: http://www.w3schools.com/tags/att_button_formtarget.asp

File: app/views/events/participants.html.erb [hunk 4]
```diff
@@ -127,8 +130,9 @@
   <% if @event.phase == :execution %>
     <%= link_to t('events.badges.print_button_label'),
                 badges_event_path(@event),
-                class: 'btn btn-default',
-                disabled: @participants.empty? %>
+                :class => 'btn btn-default',
+                :disabled => @participants.empty?,
+                :target => '_blank' %>
```
Comment (Blaidd-Drwg) [inline app/views/events/participants.html.erb]: This link doesn't need to open in a new tab, it loads an HTML page.

**Review by Blaidd-Drwg — APPROVED** [review]

**Pull request merged**